whitepapers/Tables.md

```diff
@@ -88,28 +91,59 @@ Each row corresponds to a Transformer implementation in python. (TODO: describe
+| `date_modified`         | datetime  |         |          |     | now (on update) | Datetime this row was last modified                                                                           |
+| `share_version`         |   text    |         |          |     |     UNKNOWN     | The commitish at the time this job was last run                                                               |
+| `harvester_version`     |   text    |    ✓    |          |     |   000.000.000   | Semantic version of the harvester, with each segment padded to 3 digits (e.g. '1.2.10' => '001.002.010')      |
+| `ingest_config_version` |   text    |    ✓    |          |     |   000.000.000   | Semantic version of the `IngestConfig`, with each segment padded to 3 digits (e.g. '1.2.10' => '001.002.010') |
 
 #### Other indices
 * `ingest_config_id`, `harvester_version`, `start_date`, `end_date` (unique)
 
-### TransformLog
-Log entries to track the status of a transform task
+#### Notes
+* In the future, jobs may attempt to optimize themselves by searching for jobs that have already harvested a section of it required date range
+* In the future, jobs may be merged together if they have overlapping date ranges to save on table space
+
+
+### HarvestLogRawDatum
+Through-table that links a `RawDatum` to a `HarvestLog`. (Let Django generate this)
 
-| Column                |   Type   | Indexed | Nullable | FK  | Default | Description                                                                                                |
-| :-------------------- | :------: | :-----: | :------: | :-: | :-----: | :--------------------------------------------------------------------------------------------------------- |
-| `raw_id`              |   int    |         |          |  ✓  |         | RawData to be transformed                                                                                  |
-| `ingest_config_id`    |   int    |         |          |  ✓  |         | IngestConfig used                                                                                          |
-| `transformer_version` |   text   |         |          |     |         | Semantic version of the transformer, with each segment padded to 3 digits (e.g. '1.2.10' => '001.002.010') |
-| `started`             | datetime |         |          |     |         | Time `status` was set to STARTED                                                                           |
-| `status`              |   text   |         |          |     | INITIAL | Status of the transform task, one of {INITIAL, STARTED, RESCHEDULED, SUCCEEDED, FAILED}                    |
+| Column           | Type | Indexed | Nullable | FK  | Default | Description |
+| :--------------- | :--: | :-----: | :------: | :-: | :-----: | :---------- |
+| `harvest_log_id` | int  |    ✓    |          |  ✓  |         |             |
+| `rawdatum_id`    | int  |    ✓    |          |  ✓  |         |             |
+
+
+### IngestLog
```

Comment:
Hmm, I think it makes sens for Ingest not to include Harvest (because harvesting a crop and making a pile is not part of eating it), but then we're lacking a word for the larger box...

```
 +-----------------------------------------------------------------+
 |                        ???                                      |
 |                                                                 |
 |                  +--------------------------------------------+ |
 | +---------+      |                  Ingest                    | |
 | | Harvest | ---> | Transform  --->  Regulate ---> Consolidate | |
 | +---------+      +--------------------------------------------+ |
 |                                                                 |
 +-----------------------------------------------------------------+
```